docs/components/app-screen.md

```diff
@@ -35,54 +35,15 @@ Everything considered part of the App Screen is placed into the `id=page-wrapper
 
 ## Sidebar with Navigation
 
-The first child element is the Sidebar with Navigation. This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
+The first child element of the `id=page-wrapper` is the [Sidebar with Navigation](/components/sidebar-nav/). This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
```

Comment:
It would be nice if this link reference was replaced everywhere on the page (like in the Topbar section text).

Reply:
[Sidebar with Navigation](/components/sidebar-nav/)

Reply:
not worrying about that